EDDI/MainWindow.xaml.cs

```diff
@@ -165,6 +165,9 @@ public MainWindow(bool fromVA = false)
 
             this.fromVA = fromVA;
 
+            // Increment to a new log file.
```

Comment:
I am puzzled what this achieves and how the logs get incremented under VA.

Reply:
You're right. It could be clearer. I'll drop in a commit to do a separate call in each initialization pathway rather than using MainWindow.caml.cs.